doc/api/fs.md

```diff
@@ -1007,6 +1007,39 @@ property indicating whether parent directories should be created. Calling
 `fsPromises.mkdir()` when `path` is a directory that exists results in a
 rejection only when `recursive` is false.
 
+```mjs
+import { mkdir } from 'fs/promises';
+
+try {
+  const path = new URL('./test/project', import.meta.url);
+  const createDir = await mkdir(path, { recursive: true });
+
+  console.log(`created ${createDir}`);
+} catch (err) {
+  console.error(err.message);
+}
+```
+
+```cjs
+const { mkdir } = require('fs/promises');
+const { resolve } = require('path');
+
+async function makeDirectory() {
+  try {
+    const path = resolve('./test/project/lol/hi');
```

Comment:
```suggestion
    const projectFolder = resolve('./test/project/lol/hi');
```

@trott @aduh95 if the other example is being changed, this one should be too, right?

Reply:
More specifically, I'd _really_ like to avoid drift between CJS and ESM docs so readers are familiar with what's happening.

Reply:
Yes, I agree it would make sense to change both.

Reply:
@aduh95 yep, I'll do that soon here. thanks ofr the ping.